Description: Add Makefile
 Add Makefile with install to /opt/node-webkit
 .
 node-webkit (0.3.4-0+opinsys1) quantal; urgency=low
 .
   * Initial release.
Author: Esa-Matti Suuronen <esa-matti.suuronen@opinsys.fi>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: other, https://github.com/opinsys/opinsys-debs

Index: node-webkit/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ node-webkit/Makefile	2014-04-25 12:28:34.210033644 +0000
@@ -0,0 +1,11 @@
+
+prefix ?= /usr/local
+
+build:
+	echo :\)
+
+install:
+	mkdir -p $(DESTDIR)/opt/node-webkit
+	install -o root -g root -m 755 nw $(DESTDIR)/opt/node-webkit
+	install -o root -g root -m 644 libffmpegsumo.so $(DESTDIR)/opt/node-webkit
+	install -o root -g root -m 644 nw.pak $(DESTDIR)/opt/node-webkit
